### ISSUE TITLE
edgePaddin and lazylaod breaks the carousel.

### ISSUE THREAD

**Issue by jechav** [ganlanyuan/tiny-slider#106]
Hi,

https://codesandbox.io/s/mqr949kvyx in this demo when we reach the last slide throw the error.

```js
Uncaught TypeError: Cannot read property 'querySelectorAll' of undefined.

[].forEach.call(slideItems[i].querySelectorAll('.tns-lazy-img'), function (img) {
```

**Comment by ganlanyuan** [comment]
Thanks @jechav ,
Add a fix in [this commit](https://github.com/ganlanyuan/tiny-slider/commit/89a240f9fb7b1b7c42adad22eb3b82f1c3518049).

**Comment by ganlanyuan** [comment]
Fixed in [v2.3.4](https://github.com/ganlanyuan/tiny-slider/releases/tag/v2.3.4)
